service.go

```diff
@@ -180,3 +185,29 @@ func (svc *Service) createResponse(kind int, initialEvent *nostr.Event, content
 	}
 	return resp, nil
 }
+
+func (svc *Service) hasPermission(app *App, event *nostr.Event, paymentRequest *decodepay.Bolt11) bool {
```

Comment:
I think this function should return error NIP-47 error messages than a boolean. Otherwise we can't return the correct error message back to the requesting client

Reply:
fixed in other PR